home-api-tests/src/test/java/com/softserveinc/ita/homeproject/api/tests/users/UserApiIT.java

```diff
@@ -523,24 +529,25 @@ private String getToken(String str) {
     }
 
     private ApiResponse<ReadUser> updateUserWithExistEmail() throws ApiException {
+        ReadUser savedUser = baseUserForTests;
 
-        ReadUser savedUser = createBaseUserForTests();
+        ReadUser additional = createBaseUserForTests();
 
         UpdateUser updateUser = new UpdateUser()
                 .firstName("updatedFirstName")
                 .lastName("updatedLastName")
-                .email(createBaseUserForTests().getEmail())
+                .email(additional.getEmail())
                 .password("somePassword");
 
         return userApi.updateUserWithHttpInfo(savedUser.getId(), updateUser);
```

Comment:
What does mean "savedUser" in your code (raw 542)? Maybe it should be "additional.getId"?

Reply:
Creating of additional user guarantees creating of user in DB with some credentials, so I can peek an existing email from additional user to perform this test.

Reply:
Yes, everything is correct here. I was inattentive and did not see that "savedUser" created on line 532.